site/gatsby-site/src/components/leaderboards/UniqueSubmittersLeaderboard.js

```diff
@@ -19,9 +27,12 @@ const UniqueSubmittersLeaderboard = ({ limit, className }) => {
 
   const submitters = {};
 
-  for (const report of nodes) {
+  for (const report of reports) {
+    const { incident_id: id } = incidents.find((incident) =>
+      incident.reports.includes(report.report_number)
```

Comment:
The use of `find` throughout this PR is problematic and gets worse through time.